include/views/view_sycl.hpp

```diff
@@ -304,6 +265,8 @@ using BufferMatrixView = matrix_view<ScalarT, bufferT<ScalarT>>;
  */
 template <class ScalarT>
 struct matrix_view<ScalarT, bufferT<ScalarT>> {
+  constexpr static bool vectorizable =
+      Packet_traits<ScalarT, SYCLDevice>::Supported;
   using ContainerT = bufferT<ScalarT>;
```

Comment:
No, vectorization is not yet explicitly written in this code, we will need to vectorize it once we've done with the fusion.